docs/product-installation-guide/src/main/asciidoc/run-standalone-properties-con.adoc

```diff
@@ -17,21 +24,16 @@
 If you plan to use RSA or any algorithm other than DSA, make sure you setup properly your Application Server to use Bouncy Castle JCE library.
```

Comment:
It's not in this PR, but around here in the output is the following description of a property:
Passphrase to access your Operating Systems public keystore when cloning git repositories with scp style URLs; e.g. git@github.com:user/repository.git.

Change to this:

Passphrase to access the public keystore of your Operating System when cloning git repositories with SCP style URLs. Example: git@github.com:user/repository.git.